static/js/lib/queries/auth.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { FLOW_LOGIN } from "../auth"
+
+import type { RequestConfig } from "redux-query"
+
+export default {
+  loginEmailMutation: (email: string, next: ?string): RequestConfig => ({
+    url:   "/api/login/email/",
+    body:  { email, next, flow: FLOW_LOGIN },
+    force: true
+  }),
+
+  loginPasswordMutation: (
+    password: string,
+    partialToken: string
+  ): RequestConfig => ({
+    url:   "/api/login/password/",
+    body:  { password, partial_token: partialToken, flow: FLOW_LOGIN },
+    force: true
```

Comment:
I could be wrong but I don't think these `force: true`'s are necessary. I believe they're only relevant to request-type queries (i.e.: read-only): https://github.com/amplitude/redux-query#query-configs